input/resources/valueset-common-au-recorded-sex-or-gender-source-document-jurisdiction.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ValueSet xmlns="http://hl7.org/fhir">
+    <id value="common-au-recorded-sex-or-gender-source-document-jurisdiction" />
+    <meta>
+        <profile value="http://hl7.org/fhir/StructureDefinition/shareablevalueset" />
+    </meta>
+    <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
+        <valueInteger value="0"/>
+    </extension>
+    <url value="http://terminology.hl7.org.au/ValueSet/common-au-recorded-sex-or-gender-source-document-jurisdiction" />
+    <identifier>
+        <system value="urn:ietf:rfc:3986"/>
+        <value value="urn:oid:2.16.840.1.113883.2.3.4.2.4.33"/>
+    </identifier>
+    <name value="CommonAURecordedSexOrGenderSourceDocumentType" />
```

Comment:
Will accept this for now given the FHIR JIRA change. Will raise specification feedback on this long and misleading title of a value set.